documentation/sile.sil

```diff
@@ -464,7 +464,7 @@ culpa qui officia deserunt mollit anim id est laborum.
 \line
 \end{verbatim}
 
-\noindent{}…you might not necessarily get a line break after ‘eiusmod’; you’ll
+\noindent{}…you might not necessarily get a line break after ‘tempor’; you’ll
```

Comment:
This mistake appears to be the result of my mistakenly re-wrapping text in 83e1ebb0. Given that this same `verbatim` input example is used several other places in the manual, the best fix for this is probably to revert the wrapping to the way all the other examples are rather than adjusting it here. That also makes it a little more clear in the output that the text in the example isn't just soft-wrapped.

Alternatively we could update _all_ the examples, perhaps with even slightly shorter line width in the input to make the examples even more obvious.

Reply:
Bah, this is fine. The other examples aren't `verbatium` so they are not comparable. This fix is fine.